src/redux/reducers/repos.reducer.js

```diff
@@ -0,0 +1,28 @@
+import { createSlice } from "@reduxjs/toolkit";
+import fetchRepos  from "../actions/repos.actions";
+
+const initialState = {
+    reposList: [],
+    filters:{page:1,per_page:10,visibility:"public"}
```

Comment:
`filters` can stay in the local state of the component since it is not shared with any other.
If, for some reason, we needed to have them in the store, you should dispatch new `actions` and handle them in the `reducer` to update the data from the component.

Reply:
Thank you Sir for your comments, I tried to solve the bugs and update profile and repositoriesList using redux